app/src/layouts/common/appSidebar/appSidebar.jsx

```diff
@@ -55,10 +66,28 @@ export const AppSidebar = ({
   const onClearUser = () => {
     setIsHoveredUser(false);
   };
+  const onHoverService = () => {
+    setIsHoveredService(true);
+  };
+
+  const onClearService = () => {
+    setIsHoveredService(false);
+  };
 
   const createFooterBlock = (openNavbar) => (
     <>
       <div className={cx('policy-block')} />
+      <button
+        className={cx('service-block', { 'active-tooltip': !isFAQOpened })}
```

Comment:
Why do you call it as `tooltip`?
As I see it is not has any tooltips while `isFAQOpened` is `false`, it is has a small green indicator.
So, maybe rename the class to smth like `untouched`.
Another suggestion is to use the similar name for the flag in `localStorage` - `isFaqTouched`.
As the `isFAQOpened` sounds more like a setting that should open some faq by default.